chapters/hpmor-chapter-036.tex

```diff
@@ -240,7 +240,7 @@ \chapter{Status Differentials}
 
 “Of course we know she’s smart,” said Leo Granger, starting to look offended at the child who’d had the temerity to raise his voice at their dinner table.
 
-“You don’t have the tiniest idea,” said Harry, the ice now leaking into his voice. “You think she reads a lot of books and it’s cute, right? You see a perfect report card and you think it’s good that she’s doing well in class. Your daughter is the most talented witch of her generation and the brightest star of Hogwarts, and some day, Mr and Mrs~Granger, the fact that you were her parents will be the only reason that history remembers you!”
+“You don’t have the tiniest idea,” said Harry, the ice now leaking into his voice. “You think she reads a lot of books and it’s cute, right? You see a perfect report card and you think it’s good that she’s doing well in class. Your daughter is the most talented witch of her generation and the brightest star of Hogwarts, and some day, Mr~and Mrs~Granger, the fact that you were her parents will be the only reason that history remembers you!”
```

Comment:
I don't think a non-breaking space is required here, as `Mr` is not followed by a surname. A line break after it would be OK.

Reply:
ok, reverted all "Mr~and" -> "Mr and"